modules/mediawiki/files/bin/mwscript.py

```diff
@@ -33,39 +39,35 @@ def get_commands(args: argparse.Namespace) -> CommandInfo:
                 args.arguments = False
         else:
             print(f'First argument should be a valid wiki if --extension not given DEBUG: {args.arguments[0]} / {args.extension} / {[*["all"], *validDBLists]}')
-            sys.exit(2)
+            return 2
     except IndexError:
         print('Not enough Arguments given.')
-        sys.exit(2)
+        return 2
     script = args.script
-    if not script.endswith('.php'):
-        if not args.runner:
-            print('Error: Specifiy --use-runner or --140 to enable MaintenanceRunner')
-            sys.exit(2)
-        if args.runner and not args.confirm:
-            print(f'WARNING: Please log usage of {longscripts}. Support for longscripts has not been added')
-            print('WARNING: Use of classes is not well tested. Please use with caution.')
-            if input("Type 'Y' to confirm (or any other key to stop - rerun without --140/--use-runner): ").upper() != 'Y':
-                sys.exit(2)
-    if args.runner:
+    if not script.endswith('.php') and args.norunphp:
+        print('Error: Specifiy --use-runner or --140 to enable MaintenanceRunner')
```

Comment:
--use-runner and --140 seem to be gone in this change?

Reply:
```suggestion
        print('Error: You can't use a class and specify --no-runner')
```